src/index.js

```diff
@@ -74,9 +98,71 @@ function handlebars(options) {
   const Handlebars = options.handlebars.module || require('handlebars');
   const ImportScanner = require('./ImportScanner')(Handlebars);
 
+  const validFormats = getValidFormats(options),
+    usesFragmentParsing = validFormats.element || validFormats.fragment;
+
+  const jqueryPath =
+    typeof options.jquery === 'string' ? options.jquery : validFormats.jquery ? 'jquery' : null;
+
   return {
+    resolveId(id) {
+      return id === PLUGIN_HELPER_MODULE_ID ? id : null;
+    },
+
+    load(id) {
+      if (id !== PLUGIN_HELPER_MODULE_ID) return null;
+
+      const defaultFormat = validFormats.jquery ? "'jquery'" : "'string'";
+
+      let body = `const valid = Object.create(null);\n`;
+      for (const [key, value] of Object.entries(validFormats)) {
+        body += `valid.${key} = ${value};\n`;
+      }
+      body += `const hasOwn = Object.prototype.hasOwnProperty;\n`;
+      if (usesFragmentParsing) {
+        body += `export const rangeForParsing = document.createRange();\n`;
+      }
+      if (validFormats.element) {
+        body += `function parseElement(html) {\n`;
```

Comment:
What do you think about defining these functions elsewhere and `toString`ing them here?

Reply:
Well I took that idea and ran with it, so I'm just gonna open a new PR.